package.xml

```diff
@@ -34,6 +34,10 @@
   <build_export_depend>message_runtime</build_export_depend>
   <exec_depend>message_runtime</exec_depend>
   <exec_depend>wibotic_msg</exec_depend>
+  <build_depend>actionlib</build_depend>
+  <build_depend>actionlib_msgs</build_depend>
+  <exec_depend>actionlib</exec_depend>
+  <exec_depend>actionlib_msgs</exec_depend>
 
```

Comment:
If a dependency is all of build, exec prefer using the `<depend>` tag.

Reply:
done!